tests/UiTPAS/Listeners/NotifyUdbOrganizerRequestedTest.php

```diff
@@ -176,6 +161,43 @@ public function test_it_does_not_message_approved_organizers(): void
         $this->messageBuilder->expects($this->never())
             ->method('toMessageWithOrganizer');
 
-        $this->listener->handle($event);
+        $this->listener->handleIntegrationActivationRequested($event);
+    }
+
+    public function test_it_sends_mail_when_udb_organizer_is_requested(): void
```

Comment:
Is it not the notifier we want to test here, and not the sending of mails?

Reply:
I am sorry, I do not understand your remark. can you clarify?

Reply:
I mean in the context of this test file, we are mainly interested in sending the Slack notification. You are testing an extra side effect of dispatching the event, which is also sending mails, but that is not the goal of `NotifyUdbOrganizerRequested`